_content/articles/yunnan-burma-frontier-since-the-end-of-17th-century_jianxiong-ma.md

```diff
@@ -0,0 +1,18 @@
+---
+title: "Shaping of the Yunnan-Burma Frontier by Secret Societies since the End of the 17th Century"
+authors:
+  - "Ma Jianxiong"
+external_url: "https://journals.openedition.org/moussons/538"
+drive_links:
+  - "https://drive.google.com/file/d/15J73T2aXJHyxoglIUf5zi5TshbLaECtt/view?usp=sharing"
+course: burmese
+tags:
+  - state
+  - early-modern
```

Comment:
🤔  Right now, this tag is for early modern encounters between East and West But you're right that this does belong there chronologically... Maybe add `medieval` too and we can sort it out later